components/event-processor-manager/org.wso2.carbon.event.processor.manager.commons/src/main/java/org/wso2/carbon/event/processor/manager/commons/transport/server/TCPEventServer.java

```diff
@@ -146,7 +146,7 @@ public void run() {
                         String streamId = new String(streamNameData, 0, streamNameData.length);
                         StreamRuntimeInfo streamRuntimeInfo = streamRuntimeInfoMap.get(streamId);
                         while (streamRuntimeInfo == null) {
-                            Thread.sleep(100);
+                            Thread.sleep(1000);
                             log.warn("TCP server on port :'" + tcpEventServerConfig.getPort() + "' waiting for streamId:'" + streamId + "' to process incoming events");
```

Comment:
@ramindu90 
I think better to move the warn log before sleeping so that the user gets a message before the thread goes to sleep and user would expect the thread to wait after seeing this message in the log. WDYT?